tests/integration/netapi/rest_tornado/test_app.py

```diff
@@ -292,6 +292,9 @@ def test_simple_local_async_post_no_tgt(self):
                               )
         response_obj = salt.utils.json.loads(response.body)
         self.assertEqual(response_obj['return'], [{}])
+        # TODO uncomment after deprecated (See PR #53503)
+        # self.assertEqual(response_obj['return'], ["No minions matched the target. No command was sent, no jid was assigned."])
+
```

Comment:
Also failing the linter... too many blank lines

Reply:
i fixed it. thanks